modules/src/what_cocktail.py

```diff
@@ -0,0 +1,351 @@
+                print()
+                exit()
+    else:
+        hi_matches = bye_or # works
+        print("These are your matches:")
+        for key, value in match.items():
+            if hi_matches > 1:
+                hi_matches -= 1
+                print()
+                print_nice(key, value)
+                print()
+                print("Or...")
+            elif hi_matches == 1:
+                print()
+                print_nice(key, value)
+                print()
+
+def print_nice(statement, statement2):
+    statement = str(statement)
+    statement2 = str(statement2)
+    statement = re.sub(r'[{}:\']', '', statement)
+    statement2 = re.sub(r'[{}:\']', '', statement2)
+    statement2 = re.sub(r'T?t?hin\s', '', statement2)
+    statement2 = re.sub(r'\d+\sS?s?ticks?', '',statement2)
+    statement2 = re.sub(r'(\d+\sml)', '', statement2)
+    statement2 = re.sub(r'\d+?A?a?\s\Dlices?', '', statement2)
+    statement2 = re.sub(r'\d+\s\Dedges?', '', statement2)
+    statement2 = re.sub(r'\d+\s\Dwists?', '', statement2)
+    statement2 = re.sub(r'c?C?hopped', '', statement2)
+    statement2 = re.sub(r'\d+\s\Dashe?s?', '', statement2)
+    statement2 = re.sub(r'A?a?\sS?s?prinkle', '', statement2)
+    statement2 = re.sub(r'\d+?A?a?\sP?p?inche?s?', '', statement2)
+    statement2 = re.sub(r'A?a?\d+?\sS?s?prigs?', '', statement2)
+    statement2 = re.sub(r'\d+\stsp', '', statement2)
+    statement2 = re.sub(r'\d+\sP?p?ieces?\sG?g?rated', '', statement2)
+    statement2 = re.sub(r'a?A?\d+\sS?s?plashe?s?', '', statement2)
+    statement2 = re.sub(r'T?t?o\sG?g?arnish', '', statement2)
+    statement2 = re.sub(r'T?t?o\sT?t?op\sU?u?p', '', statement2)
+    statement2 = re.sub(r'\d+.', '', statement2)
+    statement2 = re.sub(r'\s,', ',', statement2)
+    statement2 = re.sub(r'\s$', '', statement2)
+    statement2 = re.sub(r'\s\s', ', ', statement2)
+    print("%s made with %s." %(statement, statement2))
+
+def dict_lookup(ing, dict):
+    for key, value in dict.items():
+        for v in value:
+            if v == ing:
+                return True
+
+def final_print(v_state):
+    v_state = str(v_state)
+    v_state = re.sub(r'[\']', '', v_state)
+    return v_state
+
+def after_mix(final):
+    last_opt = len(final)
+    if last_opt == 0:
+        print("Mix what you got up, give it a name, pour and drank")
+        exit()
+    print()
+    print()
+    print("With the ingredients you have you have these options for a cocktail tonight:")
+    for key, value in final.items():
+        if last_opt > 1:
+            last_opt -= 1
+            print()
+            print(key)
+            print("made with these measures:")
+            print(final_print(value))
+            print()
+            print("Or...")
+        elif last_opt == 1:
+            print()
+            print(key)
+            print("made with these measures:")
+            print(final_print(value))
+            print()
+            exit()
+
+cocktails = {
```

Comment:
Would appreciate putting it either on the top of the file. Better to put it in a `.json` file and parse it in the script.

Reply:
i've put it on the top of the file for time being, i'm a beginner programmer and have not yet got to grips with .json yet

Reply:
Small modules would be easy to maintain. I suggest having a look at https://stackoverflow.com/questions/20199126/reading-json-from-a-file